internal/pkg/cli/validate.go

```diff
@@ -521,9 +537,9 @@ func validateKey(val interface{}) error {
 	if err != nil {
 		return errDDBAttributeBadFormat
 	}
-	err = dynamoTableNameValidation(*attr.Name)
+	err = dynamoAttributeNameValidation(*attr.Name)
 	if err != nil {
-		return errValueBadFormatWithPeriodUnderscore
+		return errDDBAttributeBadSize
```

Comment:
Maybe we can `return err` here since it's also possible that `err = errValueNotAString`

Reply:
Thank you for your review. I agree with you and made another commit.